ocs_ci/utility/workloads/helpers.py

```diff
@@ -21,7 +22,14 @@ def find_distro(io_pod):
     """
     for distro, pkg_mgr in DISTROS.items():
         try:
-            io_pod.exec_cmd_on_pod(f"which {pkg_mgr}", out_yaml_format=False)
+            label_dict = io_pod.get_labels()
+            if label_dict:
```

Comment:
lines 25-32 should be replaced with"
```python
            cmd = f"{pkg_mgr}"
                if not constants.DEPLOYMENTCONFIG in io_pod.get_labels():
                    cmd = "which " + cmd
            io_pod.exec_cmd_on_pod(cmd, out_yaml_format=False)
```

Reply:
this will fail if we deploy pod there are no labels available for normal pod so if condition fails with TypeError

self = <tests.manage.pv_services.test_run_io_multiple_pods.TestRunIOMultiplePodsRBD object at 0x7f19fbdbab38>

    def test_run_io_multiple_pods_rbd(self):
        """
        Run IO on multiple pods in parallel - RBD
        """
>       self.run_io_multiple_pods()

tests/manage/pv_services/test_run_io_multiple_pods.py:81: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/manage/pv_services/test_run_io_multiple_pods.py:48: in run_io_multiple_pods
    pod.run_io('fs', f'{self.pvc_size_int - 1}G')
ocs_ci/ocs/resources/pod.py:245: in run_io
    self.workload_setup(storage_type=storage_type, jobs=jobs)
ocs_ci/ocs/resources/pod.py:214: in workload_setup
    assert self.wl_obj.setup(), f"Setup for FIO failed on pod {self.name}"
ocs_ci/ocs/workload.py:67: in setup
    return self.work_load_mod.setup(**setup_conf)
ocs_ci/utility/workloads/fio.py:34: in setup
    distro = find_distro(io_pod)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

io_pod = <ocs_ci.ocs.resources.pod.Pod object at 0x7f19f856ee48>

    def find_distro(io_pod):
        """
        Find whats the os distro on pod
    
        Args:
            io_pod (Pod): app pod object
    
        Returns:
            distro (str): representing 'Debian' or 'RHEL' as of now
        """
        for distro, pkg_mgr in DISTROS.items():
            try:
                cmd = f"{pkg_mgr}"
>               if not constants.DEPLOYMENTCONFIG in io_pod.get_labels():
E               TypeError: argument of type 'NoneType' is not iterable

ocs_ci/utility/workloads/helpers.py:26: TypeError

Reply:
Will try and let you know

Reply:
@ebenahar will this be fine 

cmd = f"which {pkg_mgr}"
            logging.info(io_pod.get_labels())
            if io_pod.get_labels():
                if constants.DEPLOYMENTCONFIG in io_pod.get_labels():
                    cmd = cmd.replace("which", "")
            io_pod.exec_cmd_on_pod(cmd, out_yaml_format=False)

Reply:
@ebenahar tried above code but it fails for kind pod